GitHubConfiguration.ps1

```diff
@@ -146,6 +146,11 @@ function Set-GitHubConfiguration
         created/updated and the specified configuration changes will only remain in memory/effect
         for the duration of this PowerShell session.
 
+    .PARAMETER GitHubBaseUrl
```

Comment:
In terms of naming for this configuration, I think I have two input suggestions here:
1. Instead of saying `GitHub` as part of this name, I think it should be `Api`.
2. This doesn't seem to be a `Url`, but rather a `HostName`.

So, I think it's probably more correct to call this the `ApiHostName`.

Reply:
Suggestion implemented, I like it better 👍